src/components/InventoryV2/InventoryCellRenderer.tsx

```diff
@@ -43,29 +45,18 @@ export default function InventoryCellRenderer(props: RendererProps<TableRowType>
     );
   };
 
-  const createLinkToInventorySpeciesDetail = (iValue: React.ReactNode | unknown[]) => {
-    return (
-      <Link to={APP_PATHS.INVENTORY_ITEM_FOR_SPECIES.replace(':speciesId', row.species_id.toString())}>
-        {iValue as React.ReactNode}
-      </Link>
-    );
-  };
+  const createLinkWithQuery = (path: string, iValue: React.ReactNode | unknown[]) => (
+    <Link to={`${path}?${query.toString()}`}>{iValue as React.ReactNode}</Link>
```

Comment:
thoughts on removing the trailing `?` if there's no query?

Reply:
Good call